static/styles/zulip.css

```diff
@@ -1426,6 +1426,15 @@ td.pointer {
     }
 }
 
+/* Make the action icon on the message row
+   always visible while the popover is open */
+.has_actions_popover .actions_hover {
+    visibility: visible !important;
+    pointer-events: all !important;
+    opacity: 1 !important;
+    color: hsl(0, 0%, 73%);
```

Comment:
Does this work in the dark theme? I'd expect this `color` line to break the dark theme; I would expect just deleting this line to be correct, but haven't tested.

Reply:
I thought I tested in dark scheme, but I didn't look at the colors super carefully, so I'll leave it to @eyuelberga to comment.

Reply:
We should address this in this PR; having independent changes that we want to think about together is exactly what multiple commits in a single PR is for. I'd remove the line by amending the commit that adds it in this PR, and also add a new 3rd commit that removes the useless line in the similar reaction icon (with its own commit message explaining how you verified that the line had no effect).

Reply:
I see. I have now included a third new commit and also updated the previous one.

Reply:
Looks good, thanks!